test-vectors/presentation_exchange/validate_definition.json

```diff
@@ -0,0 +1,434 @@
+{
+  "description":"Validate definition",
+  "vectors":[
```

Comment:
Each of these vectors don't have an output, but they _do_ specify that there should/shouldn't be errors. Does that mean the validation should pass? If so, should the output be `true` to make that clearer?

Reply:
right so these are testing a function called 

```
PresentationExchange.validateDefinition()
```

so if errors is false, then this should be a valid definition. (and not throw)

if errors is true, then this should be an invalid definition json (and throw)

Reply:
@frankhinek I'd like to get your thoughts on this approach. I know that in the crypto test vectors you added (ex: [crypto_edd25519/verify.json](https://github.com/TBD54566975/web5-spec/blob/main/test-vectors/crypto_ed25519/verify.json#L4-L18)), we also have a validate/verify vector, with the output of true/false accordingly. 

I think in some languages, it makes sense to return a boolean value for validate/verify operations, in addition to producing an error. Wondering if we can capture that somehow. 

Or maybe I'm way off, and validate/verify are completely separate concepts to others. I've always thought of them as similar concepts in my head 😅

Reply:
Yea this has been the point of discussion lately, 
Across the sdks we have some verify functions that return true / false, other that throw and exception or not, and others that give a Results object

I'm capturing requirements here: and then I hope we can come to consensus on a global "Results" object (or not) to be used across all these verify functions - https://github.com/TBD54566975/web5-js/issues/386